### PR TITLE
feat: Adds example workflow callers

### LINKED ISSUE #21: Provide example workflows that use the reusable workflows
Provide examples that can simply be copied to other repositories.

### PR DESCRIPTION
**Pull request by georg-eckert-zeiss**
Closes #21

### CODE REVIEW

File: .github/workflows/create-release.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
-# Create a stable NuGet release.
+# This workflow
+# + creates a new release branch from develop,
+# + raises version numbers semantically,
+# + merges the release branch into main,
+# + sets a version tag,
+# + generates release notes from commits and contributors,
+# + creates a support branch off the last relese if necessary
```
Comment (czthiele) [inline .github/workflows/create-release.yml:7]: Typo 'relese'

File: Examples/create-release.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Calls a workflow that
+# + creates a new release branch from develop,
+# + raises version numbers semantically,
+# + merges the release branch into main,
+# + sets a version tag,
+# + generates release notes from commits and contributors,
+# + creates a support branch off the last relese if necessary
```
Comment (czthiele) [inline Examples/create-release.yml:7]: Typo 'relese'

File: Examples/create-support-release.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Calls a workflow that
+# + runs on support branches only,
+# + raises version numbers semantically,
+# + sets a version tag,
+# + generates release notes from commits and contributors
+# + and publishes a NuGet.
+#
+# This workflow must be triggered manually.
+
+name: Create & publish release from develop
```
Comment (czthiele) [inline Examples/create-support-release.yml:10]: Maybe 'Create & publish release from support branch'

**Review by czthiele — APPROVED** [review]

File: Examples/create-release.yml (hunk 2 above)
Comment (georg-eckert-zeiss) [reply Examples/create-release.yml]: Done

**Pull request merged**